CodeStarterCamp_Week4/Routine.swift

```diff
@@ -0,0 +1,36 @@
+enum RoutineError: Error {
+    case exceedFatigueError
+    case wrongValueError
+}
+
+class Routine {
+    let routineName: String = "hellRoutine"
+    let activities: [Activity] = [sitUp, squat, activeRest, longRun, activeRest]
+    
+    func countingRoutine(name: BodyCondition) throws {
+        print("루틴을 몇 번 반복할까요?")
+        guard let input = readLine(), let routineNumber = Int(input) else { throw RoutineError.wrongValueError }
+        print("--------------")
+        for number in 1...routineNumber {
+            print("\(number)번째 \(routineName)을 시작합니다.")
+            for routine in 0...activities.count - 1 {
+                guard name.fatigue < 100 else { throw RoutineError.exceedFatigueError }
+                activities[routine].action(name)
+            }
```

Comment:
요렇게 표현할 수도 있습니다. 좀 더 깔끔하죠? 😁
```suggestion
            for routine in 0..<activities.count {
                guard name.fatigue < 100 else { throw RoutineError.exceedFatigueError }
                activities[routine].action(name)
            }
```